cms_delete_content/models/website_published.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from openerp import api, fields, models, _
+
+
+class WebsitePublishedMixin(models.AbstractModel):
+    _inherit = "website.published.mixin"
+
+    cms_delete_url = fields.Char(
+        string='CMS delete URL',
+        compute='_compute_cms_delete_url',
+        readonly=True,
+    )
+
+    @api.multi
+    def _compute_cms_delete_url(self):
+        for item in self:
+            item.cms_delete_url = \
+                u'/cms/delete/{}/{}'.format(item._name, item.id)
+
+    cms_delete_confirm_url = fields.Char(
+        string='CMS delete confirm URL',
+        compute='_compute_cms_delete_confirm_url',
+        readonly=True,
+    )
+
+    @api.multi
+    def _compute_cms_delete_confirm_url(self):
+        for item in self:
+            item.cms_delete_confirm_url = \
+                u'/cms/delete/{}/{}/confirm'.format(item._name, item.id)
+
+    # customize this per-model
+    cms_after_delete_url = '/'
```

Comment:
is that a "religious" thing? :stuck_out_tongue: What's the benefit of "calling it" then? :thinking: 
Personally I found it useless in this case. To me, this sounds like a parameter, nothing more. 

In most of the cases you won't override it conditionally, since whenever you'll have the need - depending on user action - you can just append a `?redirect=`.

That said, I can move it to a method and make you sleep well tonite :stuck_out_tongue_closed_eyes: